apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx

```diff
@@ -28,6 +28,12 @@ const motionProps: ComponentProps<typeof MotionBadge> = {
   animate: { y: 0, opacity: 1, transition: { duration: 0.2 } },
 };
 
+function getDeliverableEvent(workflow: TWorkflowById): string | undefined {
```

Comment:
Let's either rename the identical functions, share the same function, or add a comment to explain the duplicated logic.